types/quick-store/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for quick-store 0.1
+// Project: https://www.npmjs.com/package/quick-store (repository is deleted)
+// Definitions by: Glenn <https://github.com/promise>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = QuickStore;
+
+interface dbContents { [key: string]: value; }
+type value = string | number | dbContents | value[] | boolean | null;
+type callback<value> = (data: value) => any;
```

Comment:
```suggestion
type callback<value> = (data: value) => void;
```
we don't have source, but those are callbacks, not predicates or reducers. There is no notion on README those can be used in any other way (like returning a bool value to impact outcome of some action, etc)